### ISSUE TITLE
希望增加 exif 标签的翻译

### ISSUE THREAD

**Issue by Zpcola** [T8RIN/ImageToolbox#981]
**Is your feature request related to a problem? Please describe.**

目前编辑exif信息工具中的exif标签显示为英文,希望能增加中文支持
![IMG_20240419_175407.jpg](https://github.com/T8RIN/ImageToolbox/assets/116626041/3667da34-b4a1-4744-9136-2ebb7c41d6ef)

不知道为什么,exif标签的信息没有存在 hosted weblat 里面

![Screenshot_2024-04-19-17-49-57-955_com.mmbox.xbrowser.jpg](https://github.com/T8RIN/ImageToolbox/assets/116626041/7ed63501-056a-467d-aab7-131c01c0f8c8)

**Comment by T8RIN** [comment]
Yes, because their names are stored as english strings, it would be hard to add localization, but i will try

**Comment by Zpcola** [comment]
> Yes, because their names are stored as english strings, it would be hard to add localization, but i will try 

感谢您的工作,imagetoolbox 是一个非常好用且美观的安卓端图片工具❤️

**Comment by T8RIN** [comment]
thanks for using!
